src/RPackage-Core/ClassDescription.extension.st

```diff
@@ -2,9 +2,9 @@ Extension { #name : #ClassDescription }
 
 { #category : #'*RPackage-Core' }
 ClassDescription >> compileSilently: code [
-	"Compile the code and classify the resulting method in the given category, leaving no trail in the system log, nor in any change set, nor in the 'recent submissions' list. This should only be used when you know for sure that the compilation will succeed."
```

Comment:
These should not be extension methods. It's just variations on existing selector familes

Reply:
Done